src/test/java/seedu/address/model/module/TutorialClassTest.java

```diff
@@ -187,4 +187,39 @@ void deleteStudent_inList_success() {
         assertTrue(tutorialClass.deleteStudent(student1));
         assertEquals(0, tutorialClass.getStudents().size());
     }
+
+    @Test
+    void deleteStudentFromTeams_success() {
+        ArrayList<Person> students = new ArrayList<>();
+
+        // Creating Person object
+        Name name1 = new Name("John");
+        Email email1 = new Email("john@example.com");
+        StudentId stuId1 = new StudentId("A0213333J");
+        Set<Tag> tags1 = new HashSet<>();
+        Person student1 = new Person(name1, email1, stuId1, tags1);
+
+        students.add(student1);
+
+        TutorialClass tutorialClass = new TutorialClass(VALID_TUTORIAL, students);
+
+        // Create teams, add student to teams, and add teams to tutorial class
+        TutorialTeam team1 = new TutorialTeam("a");
+        TutorialTeam team2 = new TutorialTeam("b");
+        team1.addStudent(student1);
+        team2.addStudent(student1);
+        tutorialClass.addTeam(team1);
+        tutorialClass.addTeam(team2);
+
+        //both teams should have 1 person
+        assertEquals(1, tutorialClass.getTutorialTeam(tutorialClass, team1).getStudents().size());
+        assertEquals(1, tutorialClass.getTutorialTeam(tutorialClass, team2).getStudents().size());
+
+        //delete the student from all teams
+        tutorialClass.deleteStudentFromTeams(student1);
+
+        //both teams should now be empty (i.e. 0 people)
+        assertEquals(0, tutorialClass.getTutorialTeam(tutorialClass, team1).getStudents().size());
+        assertEquals(0, tutorialClass.getTutorialTeam(tutorialClass, team2).getStudents().size());
```

Comment:
I think the use of magic numbers here is not very good? you can create a variable that describes the number that would be better! thanks.

Reply:
Thought the comment would be enough to explain the numbers used, but I can still see the issue.

Thanks for the check!